src/api/contracts.ts

```diff
@@ -0,0 +1,17 @@
+import { Contract } from '../interfaces';
+import { axiosInstance } from './api';
+
+const contractsApi = '/contracts';
+
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+// Implementation : https://github.com/paralink-network/paralink-node/blob/master/src/api/contracts.py
+const getAllContracts = () => axiosInstance.get<Contract[]>(`${contractsApi}`);
```

Comment:
FYI @frisitano @jbargu I've tried some of the BE Apis and had some feedback :  

1. https://github.com/paralink-network/paralink-node/blob/master/src/api/contracts.py#L47 POST & PUT should return the contract response instead of just OK ( we'd need the latest from the DB, especially in the POST to have the ID without refreshing the page or refetching all of the contracts ) 


2. Error when trying to update the status of an address, we might want to fix that
<img width="577" alt="Screenshot 2021-04-01 at 15 52 43" src="https://user-images.githubusercontent.com/6785920/113316140-eb7eb000-9305-11eb-9b13-5eab6a4222cd.png">

3. Should we be able to edit an address after it was created or saved ?  cause we don't have a "setAddress" or a modify endpoint. Also if we were to add one, let's just do it as "updateContract" endpoint and remove the status one.

4. Error when deleting. It actually works, but return a 500 as there is an error after deleting : 
<img width="860" alt="Screenshot 2021-04-01 at 15 48 21" src="https://user-images.githubusercontent.com/6785920/113316143-ec174680-9305-11eb-9f0b-1dcf06397678.png">

Reply:
Hi @ZiyedB

I have made some modifications to the API contract on this development branch - [refactor_models ](https://github.com/frisitano/paralink-node/tree/refactor_models).

This refactor modifies the API such that contracts are identified by a unique (chain, address) tuple.  

1) Under this new API you should not need any additional information returned by the API as the contract id has been removed.  

2) I am not experiencing the same issue as you.  I suspect your `.env` file may be referencing the celery broker incorrectly.  Could you ensure it contains `CELERY_BROKER_URL=amqp://guest:guest@rabbitmq:5672` please?

3) No we should not be able to update the address.  A different address would represent a different contract.

4) Fixed.

Could you take a look at the API and let me know your feedback.  I will hold fire on raising a PR to see if we need to make any modifications to the API for the front end.  In any case I still need to develop some tests for the REST endpoints.

Reply:
I forgot to mention that you will have to delete the docker volume associated with the paralink Postgres container to support the dev branch above.